mars_troughs/datapaths.py

```diff
@@ -40,21 +43,21 @@ def load_retreat_data(tmp) -> Tuple[np.ndarray, np.ndarray, np.ndarray]:
         `(n_times, n_lags)`
       lags (np.ndarray): lag values the retreats have been calculated for
         by default these are [1,2,...15,20] in millimeters
-    """
-    if tmp==1:
-        df = pd.read_csv(DATAPATHS.RETREAT)
-        times: np.ndarray = df["times"].values
-        lag_cols: List[str] = [col for col in df.columns if col.startswith("lag")]
-        lags: np.ndarray = np.array([int(col[3:]) for col in lag_cols])
-        retreats: np.ndarray = df[lag_cols].values.T
-    else:
-        df = pd.read_csv(DATAPATHS.RETREAT2)
-        times: np.ndarray = df["times"].values
-        lag_cols: List[str] = [col for col in df.columns if col.startswith("lag")]
-        lags: np.ndarray = np.array([int(col[3:]) for col in lag_cols])
-        retreats: np.ndarray = df[lag_cols].values.T
-        
-    return times, retreats, lags
+"""
+#    if tmp==1:
+#        df = pd.read_csv(DATAPATHS.RETREAT)
+#        times: np.ndarray = df["times"].values
+#        lag_cols: List[str] = [col for col in df.columns if col.startswith("lag")]
+#        lags: np.ndarray = np.array([int(col[3:]) for col in lag_cols])
+#        retreats: np.ndarray = df[lag_cols].values.T
+#    else:
+#        df = pd.read_csv(DATAPATHS.RETREAT2)
+#       times: np.ndarray = df["times"].values
+#       lag_cols: List[str] = [col for col in df.columns if col.startswith("lag")]
+#       lags: np.ndarray = np.array([int(col[3:]) for col in lag_cols])
+#       retreats: np.ndarray = df[lag_cols].values.T
+#       
+#    return times, retreats, lags
```

Comment:
I recommend deleting large commented-out blocks of code. We can always get them from git's history and it keeps the code tidy 🧹 .